native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -309,9 +321,14 @@ class GameInputProxy {
                     if (xr) {
                         xr->onRenderResume();
                     }
+
+                    auto *windowMgr = _androidPlatform->getInterface<SystemWindowManager>();
+                    auto *window = static_cast<cc::SystemWindow *>(windowMgr->getWindow(1));
```

Comment:
It may be better to rename 1 to mainWindowID

Reply:
Using a Macro?